show/main.py

```diff
@@ -219,6 +218,7 @@ def cli(ctx):
 
 if 'INCLUDE_NAT: n' not in exclude_cli_list:
     #add nat commands
+    from . import nat
```

Comment:
Could you please move changes related to NAT to a separate commit?
Also, disable nat-related unit tests, when the feature disabled
. 
Example for SNMP: https://github.com/githedgehog/sonic-buildimage/blob/fad529cc27eca5a1cbc769e3285dce38f74c34f1/rules/functions#L226